pkg/model/image_target.go

```diff
@@ -277,9 +277,10 @@ type CustomBuild struct {
 	// export $EXPECTED_REF=name:expected_tag )
 	Tag string
 
-	Fast        FastBuild
-	LiveUpdate  LiveUpdate // Optionally, can use LiveUpdate to update this build in place.
-	DisablePush bool
+	Fast             FastBuild
+	LiveUpdate       LiveUpdate // Optionally, can use LiveUpdate to update this build in place.
+	DisablePush      bool
```

Comment:
Hmm it seems like `DisablePush` is still on this struct but it has been removed as a parameter in the Tiltfile. Was that intentional?

Reply:
what makes you say it's been removed?

Reply:
Oh, I thought the changes in `CustomBuilder` were in the Tiltfile, nevermind.

Reply:
ya, fortunately we have lots of good tests for the other places it's used :smile: